wcivf/apps/elections/templates/elections/includes/_post_meta_title.html

```diff
@@ -1 +1,5 @@
-{{ object.election.name }}: The {{ person_posts|length }} candidates in {{ object.post.label }}
\ No newline at end of file
+{% if postelection.people|length %}
+{{ object.election.name }}: The {{ postelection.people|length }} candidates in {{ object.post.label }}
```

Comment:
When you have an `if` block split across lines like this, indent the text within the block e.g.:

```
{% if postelection.people|length %}
    {{ object.election.name }}: The {{ postelection.people|length }} candidates in {{ object.post.label }}
{% else %}
....
```